Orange/widgets/data/icons/SelectColumnsRandom.svg

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 16.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 width="48px" height="48px" viewBox="0 0 48 48" enable-background="new 0 0 48 48" xml:space="preserve">
+<rect x="30" y="32" fill="#FFFFFF" width="10" height="8"/>
+<g>
+	<path fill="#333333" d="M6,6v36h36V6H6z M40,14v8v1v8v9H29H19h-1H8v-8v-1v-8v-1v-8l0,0h10l0,0h1l0,0h10l0,0h1l0,0H40L40,14z"/>
+	<polygon fill="#666666" points="19,31 19,23 19,22 19,14 18,14 18,22 8,22 8,23 18,23 18,31 8,31 8,32 18,32 18,40 19,40 19,32 	
+		"/>
+	<polygon fill="#666666" points="30,14 29,14 29,22 29,23 29,31 29,32 29,40 40,40 40,31 30,31 30,23 40,23 40,22 30,22 	"/>
+	<rect x="8" y="32" fill="#FFFFFF" width="10" height="8"/>
+	<rect x="8" y="23" fill="#FFFFFF" width="10" height="8"/>
+	<polygon fill="#FFFFFF" points="8,14 18,14 18,14 	"/>
+	<rect x="8" y="14" fill="#FFFFFF" width="10" height="8"/>
+	<rect x="19" y="32" fill="#B2B2B2" width="10" height="8"/>
+	<rect x="19" y="23" fill="#B2B2B2" width="10" height="8"/>
+	<polygon fill="#B2B2B2" points="19,14 29,14 29,14 	"/>
+	<rect x="19" y="14" fill="#B2B2B2" width="10" height="8"/>
+	<rect x="30" y="23" fill="#FFFFFF" width="10" height="8"/>
+	<rect x="30" y="32" fill="#FFFFFF" width="10" height="8"/>
+	<polygon fill="#FFFFFF" points="30,14 40,14 40,14 	"/>
+	<rect x="30" y="14" fill="#FFFFFF" width="10" height="8"/>
+	<rect x="19" y="22" fill="#989898" width="10" height="1"/>
+	<rect x="19" y="31" fill="#989898" width="10" height="1"/>
+</g>
+<g>
+	<path fill="#231F20" stroke="#FFFFFF" stroke-miterlimit="10" d="M20.188,16.784h-4.769c-0.024-2.336,0.776-4.267,2.402-5.792
+		s3.768-2.288,6.425-2.288c1.711,0,3.166,0.315,4.367,0.946s2.164,1.541,2.885,2.73c0.723,1.189,1.084,2.439,1.084,3.749
+		c0,0.716-0.115,1.42-0.346,2.111c-0.23,0.692-0.596,1.401-1.092,2.129c-0.291,0.425-0.99,1.141-2.094,2.147
+		c-1.055,0.959-1.717,1.635-1.984,2.029c-0.266,0.395-0.49,0.919-0.672,1.574c-0.184,0.655-0.299,1.565-0.346,2.73h-4.933
+		l-0.018-0.819c0-1.626,0.297-3.103,0.892-4.432c0.594-1.329,1.56-2.575,2.894-3.74c1.287-1.128,2.063-1.904,2.33-2.33
+		c0.266-0.424,0.4-0.922,0.4-1.492c0-0.776-0.324-1.435-0.975-1.975c-0.648-0.54-1.52-0.81-2.611-0.81
+		c-1.116,0-2.032,0.329-2.748,0.987C20.563,14.898,20.2,15.746,20.188,16.784z M23.664,30.925c0.885,0,1.641,0.304,2.266,0.91
+		c0.625,0.607,0.938,1.341,0.938,2.202c0,0.862-0.311,1.599-0.928,2.212c-0.619,0.612-1.365,0.919-2.24,0.919
+		c-0.873,0-1.616-0.307-2.229-0.919c-0.613-0.613-0.919-1.35-0.919-2.212c0-0.861,0.303-1.595,0.91-2.202
+		C22.068,31.229,22.802,30.925,23.664,30.925z"/>

```

Comment:
Could the white border around the question mark be made a pixel or two wider?

![screenshot - 03122016 - 08 51 51 pm](https://cloud.githubusercontent.com/assets/684364/13724910/8375fef8-e894-11e5-943f-4581dd46f05d.png)